Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/PhotoUriManager.kt

```diff
@@ -26,7 +27,11 @@ import android.provider.MediaStore
 class PhotoUriManager(private val appContext: Context) {
 
     private val photoCollection by lazy {
-        MediaStore.Images.Media.getContentUri(MediaStore.VOLUME_EXTERNAL_PRIMARY)
+        if (Build.VERSION.SDK_INT > 28) {
```

Comment:
Use `Build.VERSION_CODES.P`

Reply:
Nah, the preference is not to do that now, and to just hardcover the number